### DISCUSSION TITLE
Cleaning "Issues"

### DISCUSSION THREAD

**Discussion by osholm** [silx-kit/fabio#515] [General]
Hi all,

now that I started using fabio more actually again. I have browsed around the Issues section to see if some of issues I had found were already raised. 

I seems to me that there are quite a bit of rather old issues still open and some these have little information or haven't been updated for a long time. 

Should we try to go through the old issues to clean up the list or what do you think?

Best regards,
Henning

**Comment by kif** [comment]
This looks like a good idea, especially that the next release will be flagged as "stable"
